compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/SchemaGenerator.java

```diff
@@ -141,7 +162,7 @@ private Field getField(ResourceMethodSymbol methodSymbol, List<PathSegment> list
             String deprecationReason = getDeprecationReason(methodSymbol);
             Type fieldType = getType(methodSymbol);
             Field field = new Field(list.get(0).signature(), getDescription(methodSymbol), fieldType, isDeprecated,
-                                    deprecationReason);
+                    deprecationReason);
```

Comment:
Let's keep this as earlier. check other places as well.

Reply:
Fixed